api/rest/config.go

```diff
@@ -122,6 +122,12 @@ type Config struct {
 
 	// Tracing flag used to skip tracing specific paths when not enabled.
 	Tracing bool
+
+	// HTTPLogFile is path of the file that would save HTTP API logs. If this
+	// path is empty, HTTP logs would be sent to standard output. This path
+	// should either be absolute or relative to cluster base directory. Its
+	// default value is empty.
+	HTTPLogFile string
```

Comment:
You have attached the HTTPLogFile option to the bottom. It would be better after BasicAuthCredentials. Tracing is always last.